packages/ui-components/src/Input/SelectInput.tsx

```diff
@@ -0,0 +1,312 @@
+import React, {
```

Comment:
When Joe first introduced the `SelectInput` component, we considered adding it to ui-components then. In the end we decided that this component is too specialized to what it's doing and is unlikely to be reused outside of Cockroach Cloud Console without significant refactor to make it more generic.

Therefore, I don't believe we should be adding it to our common forms here.